hail/python/test/hailtop/batch/utils.py

```diff
@@ -0,0 +1,10 @@
+from hailtop.batch_client.client import Job
+
+
+def job_logs(batch):
+    logs = {}
+    jobs = batch.jobs()
+    for j_status in jobs:
+        j = Job(batch, j_status['job_id'])
+        logs[j.id] = j.log()
+    return logs
```

Comment:
Pytest will truncate the output unless it is a string. Either convert to a string here with `str(logs)` or convert in each assert: `str(job_logs(res))`.